toolbox/__init__.py

```diff
@@ -93,6 +109,15 @@ def func():
         # UMAP legend
         self.ui.clear_button.clicked.connect(self.clear_utterances)
 
+    def set_current_wav(self, index):
+        self.current_wav = self.waves_list[self.waves_count - 1 - index]
```

Comment:
This is not working for me.
```
Exception: list index out of range
Exception: Unsupported type: 'object'
```

Steps to reproduce: Generate 16 waves, then select "15" from the dropdown menu and click "Replay"

Reply:
Should line 113 be something like this? (Haven't tested it or even thought through all the cases, but seems like it should work)
```
    self.current_wav = self.waves_list[len(self.waves_list) - (index + 1)]
```

Reply:
yeah I already just comited exactly that :P

Reply:
I am testing right now.

Reply:
Works to me.